frontend/src/components/Admin/TeamEvent/TeamEventCreditReview.tsx

```diff
@@ -39,21 +39,20 @@ const TeamEventCreditReview = (props: {
   };
 
   const rejectCreditRequest = () => {
-    ImagesAPI.deleteEventProofImage(teamEventAttendance.image).then((_) => {
-      TeamEventsAPI.deleteTeamEventAttendance(teamEventAttendance.uuid)
-        .then(() => {
-          Emitters.generalSuccess.emit({
-            headerMsg: 'Team Event Attendance Rejected!',
-            contentMsg: 'The team event attendance was successfully rejected!'
-          });
-        })
-        .catch((error) => {
-          Emitters.generalError.emit({
-            headerMsg: "Couldn't reject the team event attendance!",
-            contentMsg: error
-          });
+    ImagesAPI.deleteEventProofImage(teamEventAttendance.image);
+    TeamEventsAPI.deleteTeamEventAttendance(teamEventAttendance.uuid)
+      .then(() => {
+        Emitters.generalSuccess.emit({
+          headerMsg: 'Team Event Attendance Rejected!',
+          contentMsg: 'The team event attendance was successfully rejected!'
         });
-    });
+      })
+      .catch((error) => {
+        Emitters.generalError.emit({
+          headerMsg: "Couldn't reject the team event attendance!",
+          contentMsg: error
+        });
+      });
```

Comment:
I think that it would be better to make sure that the attendance doc gets deleted first before deleting the proof image. That way if something goes wrong with deleting the attendance doc, we haven't already accidentally deleted the attendance doc. We're better off having deleting the attendance doc and having the deletion of the image fail for whatever reason.

Reply:
Makes sense